studies/LowerCaseIntentSchemesKillSwitch.json5

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    name: 'LowerCaseIntentSchemesKillSwitch',
+    experiment: [
+      {
+        name: 'Disabled_KillSwitch',
+        probability_weight: 100,
+        feature_association: {
+          disable_feature: [
+            'LowerCaseIntentSchemes',
+          ],
+        },
+      },
+    ],
+    filter: {
+      min_version: '136.1.78.94',
```

Comment:
Please specify max_version where the issue is fixed (138.* or 137.*?)

Reply:
I don't know where it will be uplifted yet, let's do 138.* as it's already in main